backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -302,10 +307,45 @@ public RepositoryEntity getRepositoryById(Session session, RepositoryIdentificat
       CommitDAO commitDAO, SetRepository request, UserInfo userInfo, boolean create)
       throws ModelDBException, InvalidProtocolBufferException, NoSuchAlgorithmException {
     try (Session session = ModelDBHibernateUtil.getSessionFactory().openSession()) {
-      RepositoryEntity repositoryEntity;
-      final Repository repository = request.getRepository();
-      if (create) {
-        WorkspaceDTO workspaceDTO = verifyAndGetWorkspaceDTO(request.getId(), false, true);
+      RepositoryEntity repository = setRepository(session, commitDAO, request, userInfo, create);
+      return newBuilder().setRepository(repository.toProto()).build();
```

Comment:
can we have fully qualification for the builder and remove the static import?

Reply:
yes resolved